Source/FakeItEasy/Core/MethodInfoManager.cs

```diff
@@ -183,7 +184,7 @@ public override bool Equals(object obj)
             {
                 var other = (TypeMethodInfoPair)obj;
 
-                return this.Type.Equals(other.Type) && this.MethodInfo == other.MethodInfo;
+                return this.Type == other.Type && this.MethodInfo == other.MethodInfo;

```

Comment:
This was just so things looked prettier. `Type` has an `operator==`, so why not use it?